test/e2e_node/topology_manager_test.go

```diff
@@ -960,6 +964,10 @@ var _ = SIGDescribe("Topology Manager", framework.WithSerial(), feature.Topology
 	f.NamespacePodSecurityLevel = admissionapi.LevelPrivileged
 
 	ginkgo.Context("With kubeconfig updated to static CPU Manager policy run the Topology Manager tests", func() {
-		runTopologyManagerTests(f)
+		runTopologyManagerTests(f, nil)
+	})
+	ginkgo.Context("With kubeconfig's topologyOptions updated to static CPU Manager policy run the Topology Manager tests", ginkgo.Label("MaxAllowableNUMANodes"), func() {
+		allowableNUMANodes := strconv.Itoa(topologymanager.DefaultMaxAllowableNUMANodes * 2)
```

Comment:
we need to think a tiny bit more about the naming but yes, this is what I meant and the change generally LGTM. Thanks!

Reply:
how about the doubleDefaultMaxAllowableNUMANodes? Do you have a better suggestion?